### PR TITLE
WIP -- git cactus notes

### PR DESCRIPTION
**Pull request by higgins**
Here are some notes on how we could use git cactus tooling to create the notes. Don't know enough of nodegit to do this properly/quickly

@TylerBrock thoughts?

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -128,6 +130,16 @@ async function tagVersion(args) {
   return 'Done!';
 }
 
+// TODO:
+// - list all branches on upstream
+// - sort by RELEASE_PREFIX
+// - select one previous to current version
+// - git shortlog currentBranch...previousBranch
+// - generate github url to this effect
+async function createReleaseNotes(args) {
```
Comment (TylerBrock) [inline index.js]: I'm not sure if this should create the release notes (output markdown for example) but I'd definitely be ok with it showing the log between current tag and previous tag (which may be a simpler way to get at this info). Depending on branches for this info would be error prone but diff `<previous_tag>...<current_tag> would work`

Comment (higgins) [reply index.js]: cool, yeah perhaps "notes" is misleading. just some sugar to output the ```git shortlog``` between the latest releases.

i'll see if i can't find another way to get the previous tag; semver doesn't have a ```decr``` (which is probably good)

Comment (TylerBrock) [reply index.js]: Yeah at least we would know what the previous version is when you are running the tool. For example: you are doing either a tag or a cut with major/minor flag so we know how to get back to where you were... currently v1.2.3 doing a minor cut, etc

**Comment by higgins** [comment]
closing. will re-open when i shortlog-ify the output

**Pull request closed without merging**